lib/config/go_routes/go_router.dart

```diff
@@ -185,23 +188,46 @@ abstract class AppRoutes {
               ],
               redirect: loggedOutRedirect,
             ),
-            GoRoute(
-              path: 'newprivatechat',
-              pageBuilder: (context, state) => defaultPageBuilder(
-                context,
-                const NewPrivateChat(),
-              ),
-              redirect: loggedOutRedirect,
-              routes: [
-                GoRoute(
-                  path: 'newgroup',
-                  pageBuilder: (context, state) => defaultPageBuilder(
+            if (PlatformInfos.isMobile)
```

Comment:
IMO, it should be 
```suggestion
            if (responsiveUtils.isMobile && PlatformInfos.isMobile)
```

Reply:
FirstColumnInnerRoute {
   bool goAvailableInFirstColumn(context);
   // ....
}

Reply:
I can't access to context from here, it's too early. i think we should change the condition to PlatformInfos.isMobile only.

Reply:
or you stills want to support the landscape mobile, I can remove the newprivatechat goRouter completely, and use Navigator instead. But it has disadvantages:
- More memory usages (because of Navigator widget)
- Developers have to aware of goRouter and Navigator when using it to develop features in mobile devices